content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -521,33 +521,33 @@ second ensure that [keys and certificates are being delivered](#keys-and-certifi
 If everything appears to be working so far, the next step is to verify that the right [authentication policy](/docs/tasks/security/authentication/authn-policy/)
 is applied and the right destination rules are in place.
 
-## Citadel is not behaving properly {#repairing-citadel}
+## Istiod certificate management is not behaving properly {#repairing-istiod}
 
 {{< warning >}}
-Workloads with new Kubernetes service accounts can not be started when Citadel is
+Workloads with new Kubernetes service accounts can not be started when Istiod is
 disabled for maintenance since they can't get their certificates generated.
 {{< /warning >}}
 
-Citadel is not a critical data plane component. The default workload certificate lifetime is 3
-months. Certificates will be rotated by Citadel before they expire. If Citadel is disabled for
+The default workload certificate lifetime is 3
+months. Certificates will be rotated by Istiod before they expire. If Istiod certificate management is disabled for
 short maintenance periods, existing mutual TLS traffic will not be affected.
 
-If you suspect Citadel isn't working properly, verify the status of the `istio-citadel` pod:
+If you suspect Istiod isn't working properly, verify the status of the `istiod` pod:
```

Comment:
Does this procedure make sense anymore? Are these really the correct instructions for diagnosing certificate management problems? Wouldn't there be other more serious problems occurring if istiod isn't running?

cc @istio/wg-security-maintainers, PTAL

Reply:
Added security label.